frontend/src/app/helpers/skeleton-loaders/form-field-skeleton-loader/form-field-skeleton-loader.component.html

```diff
@@ -4,14 +4,15 @@
  -->
 
 <ng-container *ngIf="loading">
-  <ngx-skeleton-loader
-    class="test"
-    appearance="circle"
-    [theme]="{
-      'border-radius': '5px',
-      height: '52px',
-      width: '448px',
-      border: '1px solid white'
-    }"
-  ></ngx-skeleton-loader>
+  <div class="margin">
```

Comment:
Having this skeleton loading thingy is nice, I suppose, but isn't it a bit over the top? Does it really take that long to load the license server URL.

IMHO less is more, in this case.

Reply:
Depends on the network connection and the routing in the cluster. Locally, it loads immediately. On the server side it can take some seconds. We have some components without loading animation and it feels really "laggy". I always think that there is no license linked and after some time, the license pops up.

Reply:
And the important thing is: It keeps loading when the request fails. Otherwise, we would need to handle this case separately.